File: latincms/src/main/webapp/WEB-INF/scripts/controller/ABMComentarioController.js
latinCMS.controller("ABMComentarioController", ["$scope", "ComentarioService", ABMComentarioController]);

function ABMComentarioController($scope, ComentarioService){
	$scope.fetchAllComentarios = function(){
		ComentarioService.getAll().then(function(result){
			$scope.comentarios = result.data;
		});
	};
	
	$scope.fetchAllComentarios();
}

File: latincms/src/main/java/com/latinnet/latincms/controller/ABMComentarioController.java
package com.latinnet.latincms.controller;

import org.dozer.Mapper;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.RequestMapping;

import com.latinnet.latincms.model.dao.PostDAO;

@RequestMapping("/admin/abm/comentarios")
@Controller
public class ABMComentarioController{
    @Autowired
    private PostDAO postDAO;
    
    @Autowired
    Mapper mapper;
    
    @RequestMapping("/layout")
    public String getHomePage(){
	return "abmComentario/index";
    }
    
}
